### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by maxhaggmancmedu**
Please add my footer it looks really good!!!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <footer>
-    <div>Footer item</div>
-    <div>Footer item</div>
-    <div>Footer item</div>
-    <div>Footer item</div>
+    <div class="footer-item">Footer item</div>
+    <div class="footer-item">Footer item</div>
+    <div class="footer-item">Footer item</div>
```
Comment (Excaliderp) [inline index.html:4]: this looks good! good class

File: styles.css [hunk 2]
```diff
@@ -1 +1,9 @@
-
+footer {
+    background-color: black;
+    color: white;
+    display: flex;
+    font-size: 20px;
```
Comment (Excaliderp) [inline styles.css:5]: switch this to 16px

**Review by Excaliderp — CHANGES_REQUESTED** [review]
looking good! just a few tweaks and you're on the team!

**Review by robonexx — COMMENTED** [review]
Looks good

File: styles.css (hunk 2 above)
Comment (robonexx) [reply styles.css:5]: Ok

**Review by robonexx — APPROVED** [review]
Request approved, looking good

File: styles.css (hunk 2 above)
Comment (maxhaggmancmedu) [reply styles.css]: This is fixed now! @Excaliderp

**Review by Excaliderp — APPROVED** [review]
lokking gutt

**Pull request merged**